src/libraries/System.Runtime/tests/System/UIntPtrTests.cs

```diff
@@ -16,188 +18,210 @@ public static class UIntPtrTests
         [Fact]
         public static void Zero()
         {
-            VerifyPointer(UIntPtr.Zero, 0);
+            VerifyPointer(nuint.Zero, 0);
         }
 
         [Fact]
         public static void Ctor_UInt()
         {
             uint i = 42;
-            VerifyPointer(new UIntPtr(i), i);
-            VerifyPointer((UIntPtr)i, i);
+            VerifyPointer(new nuint(i), i);
+            VerifyPointer(i, i);
```

Comment:
I don't think this is testing the correct thing now. (And in fact it may have been broken during the earlier nint / IntPtr consolidation.) The C# compiler in netcore emits a _conv.u_ instruction, whereas the compiler in netfx would've emitted a _call (op explicit)_ instruction. This line presumably intends to test our implementation of the explicit operator. We might even need to drop down to reflection to call it now.

Reply:
> This line presumably intends to test our implementation of the explicit operator. We might even need to drop down to reflection to call it now, similar to what you've done in other test methods here. Alternatively, just kill it.

Given that this test is for `Ctor_UInt` and there is a separate test covering explicit conversions to/from, I took this as just wanting to separately validate that the constructor returns the same value as a cast and not as explicitly wanting to compare `new nuint` vs the `user-defined `nuint op_Explicit(uint)`.

It's definitely possible that it could have been trying to validate the latter instead, but the `TestExplicitCast` test is already covering this exact value anyways so it doesn't buy much to do it.

I'd be amenable to removing it if you'd prefer.